src/equations/compressible_euler_2d.jl

```diff
@@ -865,6 +957,48 @@ end
 end
 
 
+# Compute numerical flux in direction of a normal vector by rotating the velocity, 
+# computing the numerical flux in x-direction, and rotating the calculated flux back.
+@inline function (numflux::FluxPlusDissipation)(u_ll, u_rr, normal_vector::SVector, equations::CompressibleEulerEquations2D)
```

Comment:
Euler is rotationally invariant, so we can rotate the velocity vectors, apply *any* numerical flux, and rotate the result back.
Linear scalar advection is not, so we need to modify the flux to work with the advection velocity in the normal direction instead of the one saved in the equation.